### PR TITLE
Remove unranked sites

### PR DESCRIPTION
**Pull request by Carlgo11**
Sorry for the large PR! :disappointed: 
I ran the Alexa test along with Similarweb test to cross match sites with large no rank or rank above ~1M.

### CODE REVIEW

**Review by ApCoder123 — COMMENTED** [review]
A few sites have changed domains and or have rebranded and are still in the top 200K on Alexa, so should be replaced rather than deleted:
- ~~aa.net.uk -> theaa.com~~ 🤦‍♂️
- blackdesertonline.com -> playblackdesert.com
- joviafinancial.com -> jovia.org
- nordvpnteams.com -> nordlayer.com
- sharkhosting.co.uk -> verpex.com
- youracclaim.com -> credly.com

**Comment by phallobst** [comment]
> * aa.net.uk -> theaa.com

These are unrelated.

> * nordvpnteams.com -> nordlayer.com

This seems to be `vpn` vs. `remote`, so the latter probably deserves its own entry.

**Comment by Carlgo11** [comment]
I've added back the sites with new URLs. The only one without any documentation is ~Jovia~ _Verpex_ so I created an account to check if they still offered 2fa:
Screenshot 2022-02-27 at 01-06-20 Verpex – Security
Turns out they do but they don't require email verification :eyes:

**Comment by phallobst** [comment]
You mention Jovia, but the screenshot seems to be from Verpex. :thinking: 

Anyway, here are two doc pages for Jovia:
https://www.jovia.org/knowledge-base/how-do-i-enable-authenticator-apps-within-online-and-mobile-banking
https://www.jovia.org/knowledge-base/if-i-am-traveling-internationally-can-i-receive-a-one-time-passcode-via-sms-text-or-automated-voice-call-to-access-online-banking

**Comment by phallobst** [comment]
* axosinvest.com redirects to axos.com, logins are at axosbank.com
* epost.de redirects to deutschepost.de, logins are at epost.de

**Comment by Carlgo11** [comment]
added:
* playblackdesert.com
* jovia.org
* nordlayer.com
* verpex.com
* credly.com

merged:
* axios investing with axios bank

As `https://portal.epost.de/signup/welcome` and `https://portal.epost.de/mui/login` both indicate that the service is on epost.de and only the welcome page is on deutschepost.de I think the Alexa ranking still is correct and thus still removed.

**Comment by Carlgo11** [comment]
Added deutschepost.de and included epost.de within that entry.

File: entries/d/deutschepost.de.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "additional-domains": [
+      "epost.de"
+    ],
+    "contact": {
+      "facebook": "deutschepost",
+      "language": "de"
+    },
+    "keywords": [
+      "postal",
+      "email"
```
Comment (phallobst) [inline entries/d/deutschepost.de.json]: They do not actually offer an email service, even though they use "name@epost.de" style addresses. Users can send and receive documents electronically only within that system, or convert physical letters from/to electronic ones. Formerly, the entry was in `communication`, but I'm okay with using `postal` only.

Comment (Carlgo11) [reply entries/d/deutschepost.de.json]: Oh, haha :sweat_smile: 
I'll change it

File: entries/c/credly.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Credly": {
+    "domain": "credly.com",
+    "tfa": [
+      "totp"
+    ],
+    "recovery": "https://support.credly.com/hc/en-us/articles/360039330612",
```
Comment (phallobst) [inline entries/c/credly.com.json:7]:
```suggestion
    "recovery": "https://support.credly.com/hc/en-us/articles/360039330612",
    "notes": "Password reset also disables 2FA.",
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

File: entries/n/nordlayer.com.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
-  "NordVPN Teams": {
-    "domain": "nordvpnteams.com",
+  "NordLayer": {
+    "domain": "nordlayer.com",
     "tfa": [
       "totp",
       "sms"
     ],
```
Comment (ApCoder123) [inline entries/n/nordlayer.com.json:7]:
```suggestion
    ],
    "documentation": "https://help.nordlayer.com/hc/en-us/articles/360013295657",
```

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**